styles/modules/modals/_about.scss

```diff
@@ -0,0 +1,30 @@
+/* 
```

Comment:
Since this is all commented out, can we remove this file until there comes a point we need it (which may be now if you bring over any module specific styles from _layout.scss)?